lib/taiko.js

```diff
@@ -825,7 +825,9 @@ async function _click(selector, options, ...args) {
     if (!isElemAtPoint)
         throw Error(description(selector) + ' is covered by other element');
     options.x = X, options.y = Y;
-    await waitForMouseActions(options);
+    for (let count = 0; count < options.noOfClicks; count++) {
```

Comment:
guess this will also set clickCount option for input events, hence each click will be triggered with so many click conts... looks like clear test is failing for the same